.swiftlint.yml

```diff
@@ -0,0 +1,13 @@
+excluded:
+  - .build
```

Comment:
I would change this to just be
```
included:
  - Sources/
  - Tests/
```

Otherwise you are going to lint all your dependencies as well, which you probably don't want to do

Reply:
Don’t the dependencies live in .build? Excluding it seems to work

Reply:
You're right, except the special case when dependencies are open for editing. Then they are copied to the Packages folder.

Reply:
👍  I am so new to SPM I did not realize that was the case. I am going to modify the excluded to included based on @vzsg  and @0xTim's excellent advice